.travis.yml

```diff
@@ -15,4 +12,13 @@ before_script:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 
-script: "npm test"
+script:
+  - $CI_ACTION
+
+env:
+  global:
+    - TEST_SUITE=unit
+  matrix:
+    - CI_ACTION="npm run lint"
+    - CI_ACTION="npm test"
```

Comment:
The `matrix` is used to run multiple tasks in parallel http://docs.travis-ci.com/user/build-configuration/#The-Build-Matrix

we define a `$param` under script, and travis will take params from env > metrix to create tasks.
The `CI_ACTION` naming is referenced from old gaia travis.

We can even use matrix to run tests under different node.js version

```
node_js:
  - '0.12'
  - 'iojs'
  - '0.10'
```